public/app/ui/pages/detail/index.html

```diff
@@ -0,0 +1,16 @@
+<article class="detail">
+    <!-- ko if: loading -->
+    <div class="loading text-muted">
+        <img src="/images/loading.gif" alt="" title="Loading animation" />
+        Loading Library record, please wait...
+    </div>
+    <!-- /ko -->
+    <!-- ko if: displayRecord -->
+    <div class="type-header" data-bind="text: typeHeader"></div>
+    <h1>
+        <span data-bind="component: displayForLabelField()"></span>
```

Comment:
Each collection now defines, through its `settings` object in the DI container, the name of the field that is considered the "label field" for use in cases like this.  (i.e. `Title` or `ItemName`).